.github/workflows/documentation.yml

```diff
@@ -0,0 +1,52 @@
+name: DOCUMENTATION
+
+on: [push]
+
+jobs:
+
+  documentation:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: install dependencies
+      # very ugly but did not get line breaks working...
+      run: sudo apt-get update && sudo apt-get install -yq openmpi-bin libfftw3-dev python3-dev libboost-all-dev python3-h5py doxygen sphinx-common graphviz texlive-latex-base texlive-latex-extra texlive-latex-recommended ghostscript
+
+    - uses: actions/checkout@v2
+      with:
+        path: 'espressopp'
+
+    - name: build documentation
+      run: |
+        cmake -S espressopp -B espressopp-build
+        cmake --build espressopp-build --target doc
+
+    - uses: actions/checkout@v2
+      with:
+        repository: 'espressopp/espressopp.github.io.git'
+        path: 'docu'
+      if: github.ref == 'refs/heads/master'
```

Comment:
Isn't that also true if the target of a PR is master?

Maybe:
```
if: github.ref == 'refs/heads/master' && ( github.event_name == 'push' ||  github.event_name == 'schedule' )
```

Reply:
And honestly I would always do everything but the push.

Reply:
It triggers for all master branch pushes (main repo and forks). Does not trigger for PRs. Already tested.  
https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context